plugins/modem-manager/fu-plugin-modem-manager.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+#include <libmm-glib.h>
+
+#include "fu-plugin-vfuncs.h"
+
+#include "fu-mm-device.h"
+
+/* we need a new runtime version to support at+qfastboot */
+#define FU_PLUGIN_MODEM_MANAGER_REQUIRED_VERSION	"1.9.1"
```

Comment:
Rather than hardcoding this here (and also in `meson.build` eventually) why not just punt it off to meson.build as a configuration variable?  I think that the minimums should be really applying to both runtime and build time, but the only reason the runtime check exists is because of flatpak and snap talking to a daemon outside the container.